### PR TITLE
fix: add v2beta3 register resources

### PR DESCRIPTION
**Pull request by tao12345666333**
Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>

### Type of change:

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

Using register-gen tool to generate register.

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: pkg/kube/apisix/apis/config/v2beta3/register.go [hunk 1]
```diff
@@ -42,6 +42,16 @@ func addKnownTypes(scheme *runtime.Scheme) error {
...
+		&ApisixUpstream{},
+		&ApisixUpstreamList{},
+		&ApisixTls{},
+		&ApisixTlsList{},
+		&ApisixClusterConfig{},
+		&ApisixClusterConfigList{},
+		&ApisixConsumer{},
+		&ApisixConsumerList{},
+		&ApisixPluginConfig{},
+		&ApisixPluginConfigList{},
```
Comment (starsz) [inline pkg/kube/apisix/apis/config/v2beta3/register.go:54]: Can you add test case to cover this?

Comment (tao12345666333) [reply pkg/kube/apisix/apis/config/v2beta3/register.go:54]: No, I will modify how this file is maintained.

**Review by tokers — APPROVED** [review]

File: pkg/kube/apisix/apis/config/v2beta3/register.go (hunk 1 above)
Comment (starsz) [reply pkg/kube/apisix/apis/config/v2beta3/register.go]: Got it. And need to fix CI 😄

Comment (tao12345666333) [reply pkg/kube/apisix/apis/config/v2beta3/register.go]: I haven't thought of how to fix the CI problem for the time being.  In fact, I have already submitted the generated file 😅

**Review by gxthrj — APPROVED** [review]

**Pull request merged**